Ingesting EAMT archives

mjpost opened the pull request:
I'm ready to push this through now. I'm going to add one or two a day until they're ready. Supplants #965 #966.

I am tracking the status [here](https://docs.google.com/spreadsheets/d/1fpxmdV_BPwR6BQHyU9VJQxXeSOmy4__5nQCHBEviyAw/edit#gid=509047314) as I ingest these.

Status:
- [x] 2008
- [x] 2009
- [x] 2010
- [x] 2011
- [x] 2012
- [x] 2013 (none, apparently)
- [x] 2014
- [x] 2015
- [x] 2016 (already in Anthology, but user track needs splitting out)
- <strike>[ ] 2017 (??)</strike>
- <strike>[ ] 2018 (??)</strike>
- <strike>[ ] 2019 (??)</strike>

danielgildea commented on data/xml/2005.eamt.xml:
space in Žganec Gros

mjpost replied:
Done

danielgildea commented on data/xml/2005.eamt.xml:
one person: M. Romih

mjpost replied:
Done.

danielgildea commented on data/xml/1993.eamt.xml:
```diff
@@ -0,0 +1,96 @@
+      <editor><first>Petra</first><last>Steffens</last></editor>
+      <isbn>3-540-59040-4</isbn>
+    </meta>
+    <paper id="1">
```
Add introduction? would be consistent with other years.
https://link.springer.com/book/10.1007/3-540-59040-4

mjpost replied:
Ah, good catch. Thank you for your careful reviews!

mjpost replied:
Comparing that link to that of the MT Archive, it seems we are also missing the paper [Machine Translation and the Lexicon](https://link.springer.com/book/10.1007/3-540-59040-4). I'll add it.

mjpost commented:
For EAMT 2011, I included the project descriptions ([at the bottom of this page](http://www.mt-archive.info/10/EAMT-2011-TOC.htm)). I'm not sure if that was the right choice, but it doesn't seem to hurt.

mjpost commented:
Hi @danielgildea—this should now be completed, as I have moved over everything from the MT Archive. If you have a chance to look it over today or tomorrow, EAMT would be grateful (Mikel is hoping to announce on Weds).

danielgildea commented on data/xml/2011.eamt.xml:
L.

mjpost replied:
Fixed.

mjpost commented:
Awesome, thank you! I may sneak in a few UI fixes, and then will spend some time consolidating the merge commit message later today.

danielgildea commented:
I just noticed that EAMT 2015 was already present in W15.xml - now it appears twice.

mjpost commented:
It's actually two volumes—the second is user papers. It used to appear as a single "paper" in the first volume, but I split it out. I know I updated the metadata for the second volume, but I must have overwritten it. I'll push it up on emnlp-fixes.

mjpost commented:
I see. It's actually all one volume originally. So it's too bad it's displayed twice, but I think the volume titles are actually appropriate.
